### DISCUSSION TITLE
Format Text

### DISCUSSION THREAD

**Discussion by bizkeliza** [globaleaks/globaleaks-whistleblowing-software#3917] [Q&A] [answered]
Hello. 
Is it possible to change the formatting of the presentation text on the main page so that it appears justified right or left? And is it possible to add bold text?

Thank you and Happy New Year 2024

**Comment by evilaliv3** [comment, marked as answer]
You can use standard markdown that enables you to use bold, but to change the text alignment you would probably need to use some css styling.

**Reply by bizkeliza** [reply]
Thanks!!

**Reply by bizkeliza** [reply]
Do you know which file is where it is configured?
